unix-errno.opam

```
@@ -16,16 +16,11 @@ tags: ["errno" "errno.h" "errors" "unix" "syscall"]
 homepage: "https://github.com/xapi-project/ocaml-unix-errno"
 bug-reports: "https://github.com/xapi-project/ocaml-unix-errno/issues"
 depends: [
-  "ocaml" {>= "4.01.0"}
-  "dune" {>= "2.0"}
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "2.8"}
   "alcotest" {with-test}
-  "base-bytes"
-  "result"
-  "ctypes"
-]
-depopts: ["base-unix"]
-conflicts: [
-  "ctypes" {< "0.7.0"}
+  "ctypes" {>= "0.7.0"}
```

```suggestion
  "ctypes" {>= "0.12.0"}
```
per https://github.com/ocaml/opam-repository/pull/23544#discussion_r1144774665

---

thanks!